### ISSUE TITLE
🐞 Bug: In Render image container the " current view button" is too big for mobile view

### ISSUE THREAD

**Issue by upsaurav12** [CircuitVerse/CircuitVerse#4411]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In render image box the "current view button " the button is too big when we see it on the mobile view especially when we change along height as well

### Steps To Reproduce

Go to https://circuitverse.org/simulator
Click on "Download as image button "
See the "Current view button " in mobile view
![image](https://github.com/CircuitVerse/cv-frontend-vue/assets/116784047/8c78e633-b8fa-4458-bb69-8d3f48d4ef29)

### Expected Behavior

The button should not big in mobile view
![image](https://github.com/CircuitVerse/cv-frontend-vue/assets/116784047/3002ff43-eb55-49df-9e41-4f5575246b49)

### Screenshots

![image](https://github.com/CircuitVerse/cv-frontend-vue/assets/116784047/185caa01-9406-4310-9a1d-a34bf16f93b3)

### Device Information [optional]

```markdown
- OS: Windows
- Browser: Chrome
- version: 120
```

### Additional context

I am working on this issue 

### Are you working on this issue?

Yes

**Comment by upsaurav12** [comment]
i am working in this issue

**Comment by Deeppjp116** [comment]
i am working on the issue

**Comment by tanmoysrt** [comment]
Check here -
[Screencast from 26-12-23 07:17:24 PM IST.webm](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/43e3a5fb-7246-481a-a087-72b0d7b9b3ed)

**Comment by upsaurav12** [comment]
yes i checked it is working but for some mobile like s8+ it look like this 
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/ce0d220c-5738-4fcf-914c-3d0b37aa7ef8)
and also in surface duo atleast in my device

**Comment by tanmoysrt** [comment]
Ok then

**Comment by upsaurav12** [comment]
my roughly solution is like this 
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/6a332046-dd68-44e5-8dcb-c0a281a16f27)
if there is any changes in it??
ignore the border btw

**Comment by Me-Priyank** [comment]
Outdated. simulator now lived in cv-frontend-vue repo
